### PR TITLE
Launch Screen for iOS, rename generic react-native metadata

### PR DESCRIPTION
**Pull request by osdiab**
Replaces react-native default launch screen to one that looks like an empty page with the mint green header (iOS launch page is supposed to look like your app when it actually loads, to make the illusion of a fast loading app), also changes org name to Raha and renames `mobile` XCode app to Raha.

img_7649

### CODE REVIEW

**Comment by tinaroh** [comment]
How about doing a splash page? All green with our logo in the center. A lot of apps do that

File: ios/Raha.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1116,7 +1116,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastUpgradeCheck = 610;
-				ORGANIZATIONNAME = Facebook;
```
Comment (rahulgi) [inline ios/Raha.xcodeproj/project.pbxproj]: lol

Comment (tinaroh) [reply ios/Raha.xcodeproj/project.pbxproj]: lollool

**Comment by osdiab** [comment]
@tinaroh Apple explicitly says not to make it a splash page, but perhaps this is a case where nobody listens to what Apple actually wants you to do lololol. They say the purpose of a launch screen is to make it look like your app is loading faster, adn that it should only show up for fractions of a second on initial app launches and context switches; perhaps if we want to do that and our app load time is crappy then we should have a proper splash in app as well.

**Comment by osdiab** [comment]
Would you prefer that I do that?

**Comment by tinaroh** [comment]
I'd personally prefer a splash page over a blank page with just our header. But if the app loads really fast in iOS I don't care much. On Android it takes two seconds or so.

**Pull request merged**